pkg/getty/README.md

```diff
@@ -10,7 +10,7 @@ If you want a console getty, add the following to your `moby.yml`:
 ```
 services:
   - name: getty
-    image: "linuxkit/getty:e36b5e794256c2a56c000035f73a4d46f7ffb3e6"
```

Comment:
This is interesting. By doing this you have changes the tree hash of `pkg/getty` and therefore the tag which is produced if you rebuild this directory, hence invalidating this change. Lather, rinse, repeat...

(I noticed this because I did a bulk update in a PR and then my build no longer matched! I undid this change in that case).